vnext/Desktop.IntegrationTests/WebSocketResourcePerformanceTests.cpp

```diff
@@ -46,34 +50,58 @@ TEST_CLASS (WebSocketResourcePerformanceTest) { // See http://msdn.microsoft.com
   /// test must be run in isolation (no other tests running concurrently).
   ///
   TEST_METHOD(ProcessThreadsPerResource) {
-    const int resourceTotal = 50; // About 3 seconds total running time. 6 if we
-    // increase this value to 100.
+    // About 3 seconds total running time.
+    // 6, if we increase this value to 100.
+    const int resourceTotal = 50;
     const int maxWriteCount = 10;
     const int expectedThreadsPerResource = 3;
     const int startThreadCount = GetCurrentThreadCount();
 
     std::atomic_int32_t threadCount = 0;
+    bool errorFound = false;
+    string errorMessage;
+
+    auto server = std::make_shared<Test::WebSocketServer>(5556);
+    server->SetMessageFactory([](string &&message) { return message + "_response"; });
+    // TODO: Re-enable.
```

Comment:
Can we log an issue for this?

Reply:
Can and will.

Reply:
Please include the issue# in the comment. There shouldn't be TODO comments without a tracking issue.